onadata/libs/serializers/dataview_serializer.py

```diff
@@ -240,7 +241,7 @@ def get_instances_with_geopoints(self, obj):
 
         return False
 
-    def get_has_hxl_support(self, obj):
+    def get_has_hxl_support(self, obj) -> bool:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5558448)